components/log-viewer-webui/server/src/S3Manager.js

```diff
@@ -55,4 +55,16 @@ class S3Manager {
     }
 }
 
-export default S3Manager;
+/**
+ * Initializes a Fastify plugin, which decorates the application with an S3 manager at the
+ * "s3Manager" property when all plugin options are valid.
+ */
+export default fastifyPlugin(async (app, options) => {
+    const {region} = options;
+    if (null === region) {
+        return;
+    }
+
+    console.log(`Initializing S3Manager with region="${region}"...`);
+    await app.decorate("s3Manager", new S3Manager(region));
```

Comment:
Should we add another log after the decorate is done? Just for debugging purpose.
@junhaoliao

Reply:
i think for now it's fine not to log after init completion, since we don't expect constructor to hang. In cases where the constructor throws, we shall be notified for any exception messages.